build-tools/update

```diff
@@ -82,31 +81,42 @@ if (!ignoreGraphics) {
  * Update cachebuster and News
  *********************************************************/
 
-function updateIndex() {
-	// add hashes to js and css files
-	process.stdout.write("Updating hashes... ");
-	let indexContents = fs.readFileSync('index.template.html', {encoding: 'utf8'});
-	indexContents = indexContents.replace(/(src|href)="\/(.*?)\?[a-z0-9]*?"/g, function (a, b, c) {
-		let hash = Math.random(); // just in case creating the hash fails
-		try {
-			const filename = c.replace('/play.pokemonshowdown.com/', '');
-			const fstr = fs.readFileSync(filename, {encoding: 'utf8'});
-			hash = crypto.createHash('md5').update(fstr).digest('hex').substr(0, 8);
-		} catch (e) {}
+const URL_REGEX = /(src|href)="\/(.*?)(\?[a-z0-9]*?)?"/g;
 
-		return b + '="/' + c + '?' + hash + '"';
-	});
-	let indexContents2 = fs.readFileSync('preactalpha.template.html', {encoding: 'utf8'});
-	indexContents2 = indexContents2.replace(/(src|href)="\/(.*?)\?[a-z0-9]*?"/g, function (a, b, c) {
+function updateURL(a, b, c, d) {
+	c = c.replace('/replay.pokemonshowdown.com/', '/' + Config.routes.replays + '/');
```

Comment:
Not a huge fan of using multiple `replace` lines. Also, anchoring with `/` because otherwise the routes will interfere with each other). Kind of interesting to use the canonical routes as the placeholders to be replaced, but it means `index.template.html` is still valid on its own (as opposed to like `<%= Config.routes.root %>` or whatever), which is nice.